translations/validators.fr.yml

```diff
@@ -97,6 +98,8 @@ citizen_project.description.max_length: Votre texte de description est trop long
 citizen_project.message.min_length: Le message doit contenir au moins {{ limit }} caractères.
 citizen_project.emoji.max_length: Vous devez coller un emoji.
 
+citizen_project.action.invalid_finish_date: La date de fin de votre action ne peut pas dépasser le {{ limit }}.
```

Comment:
To be consistent with the wording in the project: 
```
#
# CitizenAction
#
citizen_action.invalid_finish_date: La date de fin de votre action citoyenne ne peut pas dépasser le {{ limit }}.
```

Reply:
@illusionOfParadise check `committee.event.*` keys